cms/static/sass/studio-main-v2-rtl.scss

```diff
@@ -1,14 +0,0 @@
-// ------------------------------
```

Comment:
@andy-armstrong as @bjacobel notes, the revert commit on the release branch is wiping out your second take on master, #12102. Looking into ways to avoid this.